src/renderer/components/snipping-tool.tsx

```diff
@@ -0,0 +1,167 @@
+import { ipcRenderer } from 'electron';
+import * as React from 'react';
+import { i18n } from '../../common/i18n-preload';
+
+interface IState {
+  snipImage: string;
+  height: number;
+  width: number;
+}
+
+const SNIPPING_TOOL_NAMESPACE = 'ScreenSnippet';
+
+export default class SnippingTool extends React.Component<{}, IState> {
+  private readonly eventHandlers = {
+    onDraw: () => this.draw(),
+    onHighlight: () => this.highlight(),
+    onErase: () => this.erase(),
+    onDone: () => this.done(),
+    onUndo: () => this.undo(),
+    onRedo: () => this.redo(),
+    onClear: () => this.clear(),
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      snipImage: 'Screen-Snippet',
+      height: 600,
+      width: 800,
+    };
+    this.updateState = this.updateState.bind(this);
+  }
+
+  /**
+   * Renders the Snipping tool
+   */
+  public render(): JSX.Element {
+    const { snipImage } = this.state;
+    return (
+      <div className='SnippingTool' lang={i18n.getLocale()}>
+        <header>
+          <div className='DrawActions'>
```

Comment:
How do you set the border color of a button, I don't see it in the code but I may be missing it.

Reply:
Added now.